packages/vats/src/ibc.js

```diff
@@ -240,7 +240,7 @@ export function makeIBCProtocolHandler(E, rawCallIBCDevice) {
   /**
    * @type {Store<Port, Set<PromiseRecord<ConnectionHandler>>>}
    */
-  const portToPendingConns = makeStore('Port');
+  const portToPendingConns = makeLegacyMap('Port');
```

Comment:
Please add a comment to each use of `makeLegacyMap` or `makeLegacyWeakMap` of the form
```js
// Legacy because ...
```
```suggestion
  // Legacy because it holds a raw JavaScript Set
  const portToPendingConns = makeLegacyMap('Port');
```